shared_kv_storage/db_store/db_store.cpp

```diff
@@ -98,4 +98,38 @@ void db_store_del(
     db_error::ThrowIf<db_error::RuntimeError>(presult, "db store delete failed");
 }
 // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+void db_store_csv_append(
```

Comment:
As we discussed better to you move these csv related APIs to separate file. Rather adding to same db_store.cpp. DB_store APIs are agnostic to data format.

Reply:
Sure. Will do.